llvm/include/llvm/BinaryFormat/ELFRelocs/AArch64.def

```diff
@@ -123,7 +128,7 @@ ELF_RELOC(R_AARCH64_TLSLE_LDST128_TPREL_LO12_NC,     0x23b)
 ELF_RELOC(R_AARCH64_TLSLD_LDST128_DTPREL_LO12,       0x23c)
 ELF_RELOC(R_AARCH64_TLSLD_LDST128_DTPREL_LO12_NC,    0x23d)
 ELF_RELOC(R_AARCH64_AUTH_ABS64,                      0x244)
```

Comment:
It's probably worth adding a comment here as well - now `R_AARCH64_AUTH_ABS64` is under `// TLS descriptor relocations` which does not make sense. This relocation can have both static and dynamic meaning - see it in both
https://github.com/ARM-software/abi-aa/blob/main/pauthabielf64/pauthabielf64.rst#static-data-relocations and https://github.com/ARM-software/abi-aa/blob/main/pauthabielf64/pauthabielf64.rst#auth-variant-dynamic-relocations